.rubocop.yml

```diff
@@ -46,6 +46,9 @@ Layout/HashAlignment:
 RSpec/AnyInstance:
   Enabled: false
 
+Style/RescueModifier:
+  Enabled: false
```

Comment:
I think this is for the syntax `@var = do_something rescue {}` right?
Instead of a global disable, I think is best in this case to mark the block of code with:
```
# rubocop:disable Style/RescueModifier
[...]
# rubocop:enable Style/RescueModifier
```

Those inline rescues can quickly proliferate through the code base and sometimes they become a source of headaches 😬